html/browsers/origin/cross-origin-objects/cross-origin-objects.sub.html

```diff
@@ -189,6 +192,8 @@
  * [[Delete]]
  */
 addTest(function() {
+  assert_throws(null, function() { delete C[0]; }, "Can't delete cross-origin indexed property");
```

Comment:
See https://github.com/whatwg/html/issues/1727 where I offered to do that. Felt like a separate change.